app/views/hyrax/collections/_showcase_tile.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="thumbnail">
     <div id="image-<%= index %>">
-      <%= link_to link do |body| %>
+      <%= link_to link, aria: { haspopup: true, expanded: false, controls: 'user-util-links', label: title } do |body| %>
```

Comment:
There are too many `aria-` attributes being added here. Make sure only `aria-label` is added to the link